modules/reference/pages/rpk/rpk-topic/rpk-topic-consume.adoc

```diff
@@ -10,293 +10,317 @@ The default output format `--format json` is a special format that outputs each
 record as JSON. There may be more single-word-no-escapes formats added later.
 Outside of these special formats, formatting follows the rules described below.
 
-== Format output
+== Formatting
 
-Formatting output is based on percent escapes and modifiers. Slashes can be
+Formatting is based on percent escapes and modifiers. Slashes can be
 used for common escapes:
 
-[.no-copy]
-----
-    \t \n \r \\ \xNN
-
-prints tabs, newlines, carriage returns, slashes, or hex encoded characters.p
-
-Percent encoding prints record fields, fetch partition fields, or extra values:
-
-    %t    topic
-    %T    topic length
-    %k    key
-    %K    key length
-    %v    value
-    %V    value length
-    %h    begin the header specification
-    %H    number of headers
-    %p    partition
-    %o    offset
-    %e    leader epoch
-    %d    timestamp (formatting described below)
-    %a    record attributes (formatting described below)
-    %x    producer id
-    %y    producer epoch
-
-    %[    partition log start offset
-    %|    partition last stable offset
-    %]    partition high watermark
-
-    %%    percent sign
-    %{    left brace
-    %}    right brace
-
-    %i    the number of records formatted
-----
+[cols=",",]
+|===
+|Escape |Description
```

Comment:
suggest putting the first row of the tables in bold

Reply:
this wasnt in bold because of formatting error. fixed.